tests/src/test/scala/com/nvidia/spark/rapids/SparkQueryCompareTestSuite.scala

```diff
@@ -1835,6 +1835,9 @@ trait SparkQueryCompareTestSuite extends FunSuite with Arm {
   def assumeSpark320orLater: Assertion =
     assume(VersionUtils.isSpark320OrLater, "Spark version not 3.2.0+")
 
+  def assumeSpark311Before: Assertion =
```

Comment:
Name should be `assumePriorToSpark311` to match precedent set by `assumePriorToSpark320`.